bbn-test-5/finality-providers/README.md

```diff
@@ -0,0 +1,591 @@
+stakers, and can earn commission from the staking rewards denominated in Babylon
+tokens.
+
+The finality provider toolset does not have any special hardware requirements
+and can operate on standard mid-sized machines running a UNIX-flavored operating
+system. It consists of the following programs:
+
+- _Babylon full node_: An instance of a Babylon node connecting to the Babylon
+network. Running one is not a strict requirement, but it is recommended for
+security compared to trusting a third-party RPC node.  - _Extractable One-Time
+Signature (EOTS) manager_: A daemon responsible for securely maintaining the
+finality provider’s private key and producing extractable one time signatures
+from it.  - _Finality Provider_: A daemon managing the finality provider. It
+connects to the EOTS manager to generate EOTS public randomness and finality
+votes for Babylon blocks, which it submits to Babylon through the node
+connection.
+
+The following graphic demonstrates the interconnections for parts of the above
+programs.
+
+## Install Finality Provider Binary <!-- TODO: check add in the correct tag for
+the testnet --> Download [Golang 1.21](https://go.dev/dl) 
+
+Using the go version 1.21.x (where x is any patch version like 1.21.0, 1.21.1,
+etc.) Once installed run:
+
+```shell 
+go version 
+```
+
+Subsequently clone the finality provider
+[repository](https://github.com/babylonlabs-io/finality-provider).
+
+```shell 
+git clone https://github.com/babylonchain/finality-provider.git 
+```
+
+Once the `babylon` repository is downloaded then checkout the corresponding tag
+for `bbn-testnet-5`.
+
+``` shell 
+git checkout <tag> 
+```
+
+You should now have the repository on your machine. Next navigate into the
+repository you just cloned and run the following:
+
+```shell 
+make install 
+```
+
+The above command will build and install the following binaries
+to `$GOPATH/bin`:
+
+- `eotsd`: The daemon program for the EOTS manager.  - `fpd`: The daemon program
+for the finality-provider.  - `fpcli`: The CLI tool for interacting with the
+finality-provider daemon.
+
+Now it has successfully compiled, lets check with the following command:
+
+```shell 
+eotsd 
+```
+
+This will give you a list of available actions.
+
+```shell 
+NAME:
+   eotsd - Extractable One Time Signature Daemon (eotsd).
+
+USAGE:
+   eotsd [global options] command [command options] [arguments...]
+
+COMMANDS:
+   start               Start the Extractable One Time Signature Daemon.  init
+   Initialize the eotsd home directory.  sign-schnorr        Signs a Schnorr
+   signature over arbitrary data with
+...
+```
+
+Now lets do the same with the finality provider daemon. Run:
+
+```shell 
+fpd 
+```
+
+If your shell cannot find the installed binaries, make sure `$GOPATH/bin` is in
+the `$PATH` of your shell. Usually these commands will do the job
+
+```shell 
+export PATH=$HOME/go/bin:$PATHecho 'export PATH=$HOME/go/bin:$PATH' >>
+~/.profile 
+```
+
+## Install Babylon Binary
```

Comment:
How about pointing to the babylond guide for the babylon node installation, instead of re-writing everything?